nvm.sh

```diff
@@ -3695,7 +3704,9 @@ nvm() {
             if [ $has_checked_nvmrc -ne 1 ]; then
               NVM_SILENT="${NVM_SILENT:-0}" nvm_rc_version && has_checked_nvmrc=1
             fi
-            VERSION="$(nvm_version "${NVM_RC_VERSION}")" ||:
+            provided_version="${NVM_RC_VERSION}"
+            IS_VERSION_FROM_NVMRC=1
+            VERSION="$(nvm_version "${provided_version}")" ||:
```

Comment:
```suggestion
            IS_VERSION_FROM_NVMRC=1
            VERSION="$(nvm_version "${NVM_RC_VERSION}")" ||:
```
i don't think we need to set `provided_version` if `IS_VERSION_FROM_NVMRC` is going to show a message that doesn't use it?

Reply:
`provided_version` is used/printed as part of error message when it says it's not yet installed.  Here is the example of where it is used (foo would be provided version). This was the reason why `nvm run` was not printing any error message:
```
N/A: version \"foo\" is not yet installed.

You need to run `nvm install` to install and use the node version specified in `.nvmrc`
```

Reply:
in that case:
```suggestion
            provided_version="${NVM_RC_VERSION}"
            IS_VERSION_FROM_NVMRC=1
            VERSION="$(nvm_version "${NVM_RC_VERSION}")" ||:
```